CMP1903_A1_2324/Program.cs

```diff
@@ -25,7 +25,7 @@ static void Main(string[] args)
                 {
                     int _rollValue = _diceRoller.DiceRollOnce();
```

Comment:
The variables `_diceRoller` and `_rollValue` needs to be updated to `diceRoller` and `rollValue`.
```suggestion
                    int rollValue = diceRoller.DiceRollOnce();
```